src/main/java/nwoolcan/view/warehouse/article/ArticlesInfoViewController.java

```diff
@@ -59,6 +59,17 @@ public ArticlesInfoViewController(final Controller controller, final ViewManager
         super(controller, viewManager);
     }
 
+    /*
+    @FXML
+    private void initialize() {
+        articlesSubView.getContainer().ifPresent(subViewContainer -> subViewContainer.viewChanged().addListener(newView -> {
+            if (newView == this.articlesSubView) {
```

Comment:
> not good
Oleg K.

Reply:
I have to keep it there so that it will be ready for when @Carlovan will implement that functionality and will fix #186

Reply:
update: i decided to handle the problem by myself, so i will delete them